plugins/pipelines/lib/samson_pipelines/samson_plugin.rb

```diff
@@ -0,0 +1,27 @@
+module SamsonPipelines
+  class Engine < Rails::Engine
+  end
+end
+
+Samson::Hooks.view :stage_form, 'samson_pipelines/fields'
+
+Samson::Hooks.callback :stage_permitted_params do
+  { next_stage_ids: [] }
+end
+
+Samson::Hooks.callback :after_deploy do |deploy|
+  begin
+    return unless deploy.succeeded? || deploy.stage.next_stage_id.empty?
```

Comment:
ugh, nice find... damn last minute name changes. I should have had a test for that case as well.